README.md

```diff
@@ -34,6 +34,9 @@ Below are the mutation operators available in the current version of WAF-A-MoLE.
 | Integer Encoding | `admin' OR 1=1#` ⇒ `admin' OR 0x1=(SELECT 1)#`|
 | Operator Swapping | `admin' OR 1=1#` ⇒ `admin' OR 1 LIKE 1#`|
 | Logical Invariant | `admin' OR 1=1#` ⇒ `admin' OR 1=1 AND 0<1#`|
+| Number Shuffling | `admin' OR 1=1#` ⇒ `admin' OR 2=1#`|
```

Comment:
I agree, this does seem like the case. As far as the base shuffling goes, I think that one suffers from the same problem because it won't work on all numbers at a time as well.

I wonder if these could be salvaged by having them work like that? e.g: Instead of `0=0` => `1=0`, having it be `0=0` => `1=1` and so forth.

Reply:
That makes perfect sense to me!
Even more: it could be generalized further by including more than integers. E.g., `0=0` => `'a'='a'`

Reply:
So I managed to make it work for `=` and ` LIKE ` cases, but I'm thinking about situations where we might have something like `1<2`. Should the regex take these into cases account as well or would replacing both 1 and 2 with a random replacement like previously still be okay?

Reply:
For the sake of this PR alone, I would just leave the `=` and `LIKE` cases, which are simpler to implement.

Cases like `1<2` are really interesting, for example we could add N to both sides (e.g., `2<3` or `8<9`), but I would add it as a separate mutation.

The reason is that, for equality, we can also swap with strings, while comparison requires further reasoning. For example, it is really interesting to check if some SQL dialects correctly implement `'a'<'b'` (as in ASCII ordering), but I would leave it as a separate case.